draft-irtf-cfrg-frost.md

```diff
@@ -932,24 +932,23 @@ operation can be performed.
 
 ~~~
   Inputs:
+  - s, a group secret chosen uniformly at random, such as by performing G.RandomScalar()
```

Comment:
```suggestion
  - s, a group secret that MUST be derived from at least `Ns` bytes of entropy
```

Reply:
Where is 'Ns' defined?

Reply:
It's the length of a serialized scalar, defined as part of the group.